views/admin/popup_duplicate.view.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+\Nos\I18n::current_dictionary('noviusos_form::common', 'nos::common');
+$id = $uniqid = uniqid('form_');
+?>
+<form class="fieldset standalone" id="<?= $id ?>">
+    <p>
+        <strong>
+            <?= __('Duplication target :') ?>
+        </strong>
+    </p>
+    <?php foreach ($contexts_list as $context): ?>
+        <label>
+            <?php
+            $site = \Nos\Tools_Context::site($context);
+            ?>
+            <input type="radio" name="duplicate_context"
+                   value="<?= $context ?>" <?= $item->form_context === $context ? 'checked' : '' ?> />
+            <?= \Nos\Tools_Context::contextLabel($context, array('short' => false,)) ?>
+        </label>
+        <br/>
+    <?php endforeach; ?>
+    <p style="margin: 1em 0;">
+        <button type="submit" class="ui-priority-primary ui-state-default">
+            <?= __('Duplicate') ?>
+        </button>
+        <span>
+            <?= __('or') ?>
+        </span>
+        <a href="#">
```

Comment:
In Novius OS the *Cancel* button should always be a link, that's a choice that was made by the UX team.

And it's not necessarily a bad choice : http://www.lukew.com/ff/entry.asp?571

However it's not the right place to debate this.